contrib/bsgm/__init__.py

```diff
@@ -1,2 +1,3 @@
 from vxl import vgl, vil, vpgl
+import vxl.contrib.bpgl.algo
```

Comment:
Why is this unused import here?

Usually when we import so that something is in the namespace it looks like [this](https://github.com/VisionSystemsInc/pyvxl/blob/master/vpgl/__init__.py#L2)